Packs/ThreatGrid/Playbooks/playbook-Detonate_File_-_ThreatGrid.yml

```diff
@@ -1,24 +1,24 @@
-id: detonate_file_-_threatgrid
+id: 06f52f43-59cb-4a1e-8492-5ecd612302a7
```

Comment:
Please undo this change, the `name` value must be identical to the ID. 
The SDK should catch it (opened an issue), but please fix it here (return the ID to be identical to the name)